apps/frontend/src/components/auth/ForgotPassword/useForgotPassword.ts

```diff
@@ -25,12 +28,14 @@ export const useForgotPassword = () => {
     values: ForgotPasswordFormValues,
     { setErrors }: FormikHelpers<ForgotPasswordFormValues>,
   ) => {
-    const score = await verifyRecaptcha('forgotPassword');
-    if (score <= 0.5) {
-      setErrors({
-        email: 'Sorry, Google Recaptcha has detected you as a bot',
-      });
-      return;
+    if (env.reCaptcha.secretKey && env.reCaptcha.siteKey) {
+      const score = await verifyRecaptcha('forgotPassword');
```

Comment:
move this check inside `verifyRecaptcha` function so it always return `score=1` if no secretKey && no siteKey provided.